### PR TITLE
Default config tests

### PR DESCRIPTION
**Pull request by MKashevsky**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -8,21 +8,83 @@
...
+
+
+@pytest.fixture(scope='session')
+def witness(accounts):
+    for acc in accounts:
+        if acc.is_witness():
+            return acc
+
+
+@pytest.fixture(scope='session')
```
Comment (IvanRomanovsky) [inline tests/conftest.py]: remove this fixture

File: tests/conftest.py [hunk 2]
```diff
@@ -8,21 +8,83 @@
...
+        accounts.append(create_acc(
+            test_account_name_mask.format(i + 1),
+            "10.000000000 SCR", "50.000000000 SP"
+        ))
+
+    return accounts
+
+
+@pytest.fixture(scope='session')
+def witness(accounts):
```
Comment (IvanRomanovsky) [inline tests/conftest.py]: remove this fixture

File: tests/conftest.py [hunk 3]
```diff
@@ -8,21 +8,83 @@
...
+}
+
 
 def pytest_addoption(parser):
     parser.addoption('--target', action='store', default=SCORUMD_BIN_PATH, help='specify path to scorumd')
     parser.addoption('--image', action='store', default=DEFAULT_IMAGE_NAME, help='specify image for tests run')
     parser.addoption('--use-local-image', action='store_false', help='dont rebuild image')
 
 
+def create_acc(
```
Comment (IvanRomanovsky) [inline tests/conftest.py]: hide create account logic into genesis module

File: tests/test_single_node.py [hunk 4]
```diff
@@ -19,22 +18,44 @@ def test_block_production(wallet: Wallet, node: Node):
...
+    ]
+
+    expected_total_supply = Amount()
+
+    for r in records:
+        expected_total_supply = expected_total_supply + Amount(genesis[r])
 
     assert Amount(info['total_supply']) == expected_total_supply
-    assert wallet.get_account_scr_balance(initdelegate.name) == Amount('80.000000000 SCR')
+    assert wallet.get_account_scr_balance(creator) == Amount('80.000000000 SCR')
```
Comment (IvanRomanovsky) [inline tests/test_single_node.py]: add balance check for all account in genesis

File: tests/conftest.py (hunk 1 above)
Comment (MKashevsky) [reply tests/conftest.py]: removed

File: tests/conftest.py (hunk 2 above)
Comment (MKashevsky) [reply tests/conftest.py]: removed

File: tests/conftest.py (hunk 3 above)
Comment (MKashevsky) [reply tests/conftest.py]: done

File: tests/test_single_node.py (hunk 4 above)
Comment (MKashevsky) [reply tests/test_single_node.py]: added

**Review by IvanRomanovsky — APPROVED** [review]

**Pull request merged**